### PR TITLE
New package: Golizza.RMHCentralServer version 3.50.13

### PR DESCRIPTION
**Pull request by Golizza**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/131894)

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
Since they have both server and client, the better PackageIdentifier would be `Golizza.RMHCentral.Server` or, since it is a GitHub release, `Golizza.RMHFlash.Central.Server`

**Comment by stephengillie** [comment]
Hi @Golizza,

> Since they have both server and client, the better PackageIdentifier would be `Golizza.RMHCentral.Server` or, since it is a GitHub release, `Golizza.RMHFlash.Central.Server`

I agree that this would be a better way to organize these packages. What do you think?

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

File: manifests/g/Golizza/RMHCentralServer/3.50.13/Golizza.RMHCentralServer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Golizza.RMHCentralServer
+PackageVersion: 3.50.13
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{AE562EE0-39CE-447B-A965-B74D5B70EBEF}'
```
Comment (stephengillie) [inline manifests/g/Golizza/RMHCentralServer/3.50.13/Golizza.RMHCentralServer.installer.yaml:8]:
```suggestion
ProductCode: '{AE562EE0-39CE-447B-A965-B74D5B70EBEF}'
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
```

**Comment by stephengillie** [comment]
Manual Validation ended with:
> Description: A .NET application failed. Application: RMH.Central.Communication.Central.Assistant.exe Path: C:\Program Files (x86)\Retail Hero\RMH Central Flash Server\RMH.Central.Communication.Central.Assistant.exe Message: You must install or update .NET to run this application.  App: C:\Program Files (x86)\Retail Hero\RMH Central Flash Server\RMH.Central.Communication.Central.Assistant.exe Architecture: x64 Framework: 'Microsoft.AspNetCore.App', version '6.0.0' (x64) .NET location: C:\Program Files\dotnet\  No frameworks were found.  Learn about framework resolution: https://aka.ms/dotnet/app-launch-failed  To install missing framework, download: https://aka.ms/dotnet-core-applaunch?framework=Microsoft.AspNetCore.App&framework_version=6.0.0&arch=x64&rid=win10-x64  

(Automated response - build 489.)

**Comment by stephengillie** [comment]
With both dependencies, it gives a different message:
image

**Comment by Golizza** [comment]
thank you @stephengillie. I will close this PR and submit another later.

**Pull request closed without merging**